File: Algoritmos Traduzidos/Lua/Algoritmos(Paulo Simoes)/ex4.lua
--EXERCICIO 4: Faça um algoritmo que o usuário informa um número
--			   e o algoritmo verifica se ele é um número triangular.
--Exemplo: 24 = 2 x 3 x 4


num1=1
local p

io.write'Digite um número:'

num = io.read()

p = num1*(num1+1)*(num1+2)

while tonumber(p)<tonumber(num) do

	num1 = num1+1
	p = num1*(num1+1)*(num1+2)

end

if tonumber(p)==tonumber(num) then

	print'Triangular'

else

	print'Não Triangular'
end

File: Algoritmos Traduzidos/Python/Algoritmos traduzidos[Paulo]/ex4.py
Num1 = 1
Num = input ('Digite um numero:')
P = int(Num1)*(int(Num1)+1)*(int(Num1)+2)

while int(P)<int(Num):
    Num1 = int(Num1)+1
    P = int(Num1)*(int(Num1)+1)*(int(Num1)+2)
    
if int(P) == int(Num):
     print ('Triangular')
else:
    print (' Não Triangular')
   
